app/views/linkingRequestSubmitted.scala.html

```diff
@@ -20,7 +20,7 @@
 @import uk.gov.hmrc.govukfrontend.views.html.components._
 @this(govukPanel: GovukPanel, govukButton : GovukButton, mainLayout: views.html.mainLayout)
 
-@(model: RequestSubmittedVM)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(model: RequestSubmittedVM, localAuthorityReference: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
```

Comment:
Just realised, it may be neater for me to update the RequestSubmittedVM to contain the localAuthorityReference and pass it through that way... what do you guys think worth the refactor?

Reply:
sounds sensible :thu